### ISSUE TITLE
Problems using core-js inside Devtool

### ISSUE THREAD

**Issue by galadriel2143** [zloirock/core-js#230]
I'm trying to use a node.js app inside an Electron-based debugger, devtool. Devtool has a command line switch, --no-browser-globals, which hides the window, document, and navigator variables from running scripts. The issue I'm running into is that the modules/_html script does this:

`module.exports = require('./_global').document && document.documentElement;`

The left hand side of the script succeeds, because core-js uses this black magic to grab the window object:

`Function('return this')();`

But the right hand side fails because it references the document variable. If I change the code to:

`module.exports = require('./_global').document && require('./_global').document.documentElement;`

It works properly.

**Comment by Kovensky** [comment]
w.r.t. that black magic: it's actually the _only_ reliable way of getting access to the global object, unless you forbid `eval`. As was demonstrated by this example 😆 

But yes, it would probably be better if no globals are ever assumed to be available outside of `require('./_global')`.

**Comment by pcdude2143** [comment]
Is there any way to expedite this getting fixed? I've already tracked down the issue and have a solution, it just needs to be implemented. Should I create a PR?

**Comment by zloirock** [comment]
If
```js
var global = require('./_global').document;
module.exports = document && document.documentElement;
```
will fix this issue completely - feel free add a PR.

**Comment by Kovensky** [comment]
It will fix it so the `document` is found in the "real" global environment, but it still defeats the point of `--no-browser-globals`. I guess it's good enough to at least prevent a crash...

**Comment by pcdude2143** [comment]
Yeah. It probably would be better if Devtool would correctly hide the browser globals, so that even the weird eval trick that core-js uses would return undefined, but fixing that sounds more complicated...

**Comment by zloirock** [comment]
Should be fixed in `2.5.0`.
